components/camel-zeebe/src/main/java/org/apache/camel/component/zeebe/ZeebeConsumer.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.component.zeebe;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import io.camunda.zeebe.client.api.worker.JobClient;
+import io.camunda.zeebe.client.api.worker.JobHandler;
+import io.camunda.zeebe.client.api.worker.JobWorker;
+import org.apache.camel.AsyncCallback;
+import org.apache.camel.CamelException;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.camel.component.zeebe.internal.OperationName;
+import org.apache.camel.component.zeebe.model.JobWorkerMessage;
+import org.apache.camel.support.DefaultConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ZeebeConsumer extends DefaultConsumer {
+    private static final Logger LOG = LoggerFactory.getLogger(ZeebeConsumer.class);
+
+    private final ZeebeEndpoint endpoint;
+
+    private JobWorker jobWorker;
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    public ZeebeConsumer(ZeebeEndpoint endpoint, Processor processor) throws CamelException {
+        super(endpoint, processor);
+        this.endpoint = endpoint;
+    }
+
+    @Override
+    protected void doStart() throws Exception {
+        super.doStart();
+
+        final OperationName operationName = getEndpoint().getOperationName();
+        switch (operationName) {
+            case REGISTER_JOB_WORKER:
+                if (getEndpoint().getJobKey() == null) {
+                    LOG.error("Missing JobKey");
+                    throw new CamelException("Missing JobKey");
+                }
```

Comment:
I think `ObjectHelper.notNull` would do the trick here. That's what we normally use.